src/org/labkey/test/TestProperties.java

```diff
@@ -198,6 +198,24 @@ public static boolean isTrialServer()
         return "true".equals(System.getProperty("webtest.server.trial"));
     }
 
+    /**
+     * Parses system property 'webtest.server.startup.timeout' to determine maximum allowed server startup time.
+     * If property is not defined or is not an integer, it defaults to 60 seconds.
+     * @return Maximum number of seconds to wait for server startup
+     */
+    public static int getServerStartupTimeout()
+    {
+        String property = System.getProperty("webtest.server.startup.timeout");
```

Comment:
Any value in doing this?
`System.getProperty("webtest.server.startup.timeout", 60)`

Reply:
`System.getProperty` only deals with Strings. We could do `System.getProperty("webtest.server.startup.timeout", "60")` but we'd still need to handle the `NumberFormatException` and use the default there as well.